proto/rig/api/v1/capsule/service.proto

```diff
@@ -229,10 +229,13 @@ message DeployRequest {
   bool force = 3;
   string project_id = 4;
   string environment_id = 5;
+  bool dry_run = 6;
+  uint64 based_on_rollout_id = 7;
```

Comment:
```
// If not zero, this will constrain the rollout only to be created if the currently running rollout matches this
// identifier. If this check fails, the request will return an `Aborted` error.
uint64 current_rollout_id = 7;
```

?

Reply:
LGTM